tests/backup/pages/restorePage.js

```diff
@@ -17,6 +18,6 @@ module.exports = {
   waitForRestoreSuccess(backupName) {
     I.amOnPage(this.url);
     I.waitForVisible(this.elements.backupStatusByName(backupName), 180);
-    I.waitForText('Success', 30, this.elements.backupStatusByName(backupName));
```

Comment:
same issue, can you wait for element before checking attributes.

Reply:
the comment is for the removed line, wait is a line earlier - line 20 wait, line 21 - check